src/main/java/org/apache/pulsar/manager/PulsarApplicationListener.java

```diff
@@ -0,0 +1,92 @@
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * PulsarApplicationListener do something after the spring framework initialization is complete.
+ */
+@Component
+public class PulsarApplicationListener implements ApplicationListener<ContextRefreshedEvent> {
+
+    private static final Logger log = LoggerFactory.getLogger(PulsarApplicationListener.class);
+
+    private final EnvironmentsRepository environmentsRepository;
+
+    @Value("${environment.default.name}")
+    private String environmentDefaultName;
+
+    @Value("${environment.default.broker}")
+    private String environmentDefaultBroker;
+
+    @Value("${backend.jwt.token}")
+    private String pulsarJwtToken;
+
+    public PulsarApplicationListener(EnvironmentsRepository environmentsRepository) {
+        this.environmentsRepository = environmentsRepository;
+    }
+
+    @Override
+    public void onApplicationEvent(ContextRefreshedEvent event) {
+        log.info("Start onApplicationEvent");
+        Page<EnvironmentEntity> environmentEntities = environmentsRepository
+                .getEnvironmentsList(1, 1);
+        if (environmentEntities.getResult().size() <= 0) {
+            Optional<EnvironmentEntity> environmentEntityOptional = environmentsRepository
+                    .findByName(environmentDefaultName);
+            if (environmentDefaultName != null
+                    && environmentDefaultBroker != null
+                    && environmentDefaultName.length() > 0
+                    && environmentDefaultBroker.length() > 0
+                    && !environmentEntityOptional.isPresent()) {
+                Map<String, String> header = Maps.newHashMap();
+                header.put("Content-Type", "application/json");
+                if (StringUtils.isNotBlank(pulsarJwtToken)) {
+                    header.put("Authorization", String.format("Bearer %s", pulsarJwtToken));
+                }
+                String httpTestResult = HttpUtil.doGet(environmentDefaultBroker + "/metrics", header);
+                if (httpTestResult != null) {
+                    EnvironmentEntity environmentEntity = new EnvironmentEntity();
+                    environmentEntity.setBroker(environmentDefaultBroker);
+                    environmentEntity.setName(environmentDefaultName);
+                    environmentsRepository.save(environmentEntity);
+                    log.info("The default environment set successful");
+                } else {
+                    log.error("The default environment configuration is incorrect, please check " +
+                            "configuration environment.default.name and environment.default.broker");
+                    System.exit(-1);
+                }
+            } else {
+                log.warn("The default environment already exists.");
+            }
+        }
+        log.info("The environment already exists.");
```

Comment:
change the debug level to debug otherwise this is annoying after an environment is added.

Reply:
```suggestion
        log.debug("Environments already exist");
```